### PR TITLE
Feature/blueprint extensions

### PR DESCRIPTION
**Pull request by sihu**

### CODE REVIEW

File: app/blueprints/activity_blueprint.rb [hunk 1]
```diff
@@ -6,4 +6,8 @@ class ActivityBlueprint < Blueprinter::Base
   association :activity_category, blueprint: ActivityCategoryBlueprint
   association :activity_executions, blueprint: ActivityExecutionBlueprint
   fields :label_in_database
+
+  field :stufen do |activity|
+    activity&.stufen&.map(&:code)
```
Comment (sihu) [inline app/blueprints/activity_blueprint.rb:11]: @diegosteiner what do you prefer for the calculations? ids or codes?

File: app/blueprints/unit_blueprint.rb [hunk 2]
```diff
@@ -8,4 +8,18 @@ class UnitBlueprint < Blueprinter::Base
   view :with_unit_activities do
     association :unit_activities, blueprint: UnitActivityBlueprint
   end
+
+  field :week_nr do |unit|
```
Comment (sihu) [inline app/blueprints/unit_blueprint.rb:12]: might be easier to handle

File: app/blueprints/unit_blueprint.rb [hunk 3]
```diff
@@ -8,4 +8,18 @@ class UnitBlueprint < Blueprinter::Base
...
+    case unit.week
+    when /Erste|Première|Prima/
+      1
+    when /Zweite|Deuxième|Seconda/
+      2
+    end
+  end
+
+  field :district_nr do |unit|
+    unit.district&.scan(/\d*/)&.join('')&.to_i
```
Comment (sihu) [inline app/blueprints/unit_blueprint.rb:23]: same here

File: app/models/field.rb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class Field < ApplicationRecord
-  belongs_to :spot
+  belongs_to :spot, inverse_of: :fields
```
Comment (sihu) [inline app/models/field.rb:4]: some rubocop fixes

**Review by sihu — COMMENTED** [review]
some comments

**Review by diegosteiner — APPROVED** [review]

**Pull request merged**